templates/default/keystore.sh.erb

```diff
@@ -5,8 +5,8 @@
 set -e
 
 cd <%= @directory %>
-NODE_HOSTNAME=<%= node['hostname'] %>
-NODE_FQDN=<%= node['fqdn'] %>
+NODE_HOSTNAME=<%= @hostname %>
```

Comment:
`NODE_HOSTNAME` doesn't seem to be used. I guess it can be removed.

Reply:
Actually, they are used in the name of the truststore and keystore files.